tfhe/src/integer/server_key/radix_parallel/tests_unsigned/mod.rs

```diff
@@ -59,6 +45,54 @@ pub(crate) const MAX_VEC_LEN: usize = 25;
 #[cfg(tarpaulin)]
 pub(crate) const MAX_VEC_LEN: usize = 5;
 
+/// Returns th number of loop iteration within randomized tests
+///
+/// The bigger the number of bits bootstrapped by the input parameters, the smaller the
+/// number of iteration is
+pub(crate) const fn nb_tests_for_params(params: PBSParameters) -> usize {
+    let full_modulus = params.message_modulus().0 * params.carry_modulus().0;
+
+    if cfg!(tarpaulin) {
+        // Use lower numbers for coverage to ensure fast tests to counter balance slowdown due to
+        // code instrumentation
+        1
+    } else {
+        // >= 8 bits (4_4)
+        if full_modulus >= 1 << 8 {
+            return 5;
+        }
+
+        // >= 6 bits (3_3)
+        if full_modulus >= 1 << 6 {
+            return 15;
+        }
+
+        30
+    }
+}
+
+/// Smaller number of loop iteration within randomized test,
+/// meant for test where the function tested is more expensive
+pub(crate) const fn nb_tests_smaller_for_params(params: PBSParameters) -> usize {
+    let full_modulus = params.message_modulus().0 * params.carry_modulus().0;
+
+    if cfg!(tarpaulin) {
+        1
+    } else {
+        // >= 8 bits (4_4)
+        if full_modulus >= 1 << 8 {
+            return 2;
```

Comment:
Why not 1?

Reply:
I Know some test splits the number of tests, so I wanted a value bigger than one in hope we still have a somewhat ok coverage for 4_4

Reply:
I see
IMO it would be cleaner to have these split tests do a number of tests equal to `2*nb_tests` rather `2*(nb_tests/2)`

Reply:
Btw for tarpaulin, 1 is not ok just above as we may not take some paths